#!/usr/bin/python3


def raise_exception_msg(message=""):
    '''raise a error with message'''

    raise NameError(message)
